sql/field.h

```diff
@@ -1604,7 +1604,7 @@ class Field: public Proto_field
   friend class Copy_field;
   friend class Item_avg_field;
   friend class Item_std_field;
-  friend class Item_sum_num;
+  friend class ItemSumNum;
```

Comment:
Do not change the code of Item in SQL folder.  
Do not modify MySQL code as less as possible.